src/Smuggler/Anns.hs

```diff
@@ -0,0 +1,16 @@
+module Smuggler.Anns
+       ( removeAnnAtLoc
+       ) where
+
+import Language.Haskell.GHC.ExactPrint (AnnKey (..), Annotation, Anns)
+import SrcLoc (SrcSpan (RealSrcSpan), srcSpanStartCol, srcSpanStartLine)
+
+import qualified Data.Map.Strict as Map
+
+removeAnnAtLoc :: Int -> Int -> Anns -> Anns
+removeAnnAtLoc line col = Map.filterWithKey matchKey
```

Comment:
You can write this function as `filterWithKey $ \k _ -> matchKey k` to avoid having extra argument in `matchKey` function.